src/Sylius/Bundle/CoreBundle/Form/Type/Rule/ShippingCountryConfigurationType.php

```diff
@@ -38,11 +38,9 @@ public function __construct($countryClass)
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('country', 'sylius_entity_to_identifier', [
+            ->add('country', 'sylius_country_code_choice', [
                 'label' => 'sylius.form.rule.shipping_country_configuration.country',
                 'empty_value' => 'sylius.form.country.select',
-                'class' => $this->countryClass,
```

Comment:
Is the `countryClass` field used anywhere, but here?
And if not it'd be nice to remove it and get rid of the constructor as well.